utils/opencas.py

```diff
@@ -353,7 +380,20 @@ def check_recursive(self):
                 raise ValueError('Recursive configuration detected')
 
         def to_line(self):
-            return '{0}\t{1}\t{2}\n'.format(self.cache_id, self.core_id, self.device)
+            ret = "{0}\t{1}\t{2}".format(self.cache_id, self.core_id, self.device)
+            if len(self.params) > 0:
```

Comment:
Would it be possible to remove this ` if ` statement?

Reply:
Done